Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-13: To provide and maintain a Navy;
+12: To exercise exclusive Legislation in all Cases whatsoever, over such District (not exceeding ten Miles square) as may, by Cession of particular States, and the Acceptance of Congress, become the Seat of the Government of these United States, and to exercise like Authority over all Places purchased by the Consent of the Legislature of the State in which the Same shall be, for the Erection of Forts, Magazines, Arsenals, dock-Yards, and other needful Buildings;
 
-14: To make Rules for the Government and Regulation of the land and naval Forces;
-
-15: To provide for calling forth the Militia to execute the Laws of the Union, suppress Insurrections and repel Invasions;
-
-16: To provide for organizing, arming, and disciplining, the Militia, and for governing such Part of them as may be employed in the Service of the United States, reserving to the States respectively, the Appointment of the Officers, and the Authority of training the Militia according to the discipline prescribed by Congress;
-
-17: To exercise exclusive Legislation in all Cases whatsoever, over such District (not exceeding ten Miles square) as may, by Cession of particular States, and the Acceptance of Congress, become the Seat of the Government of the United States, and to exercise like Authority over all Places purchased by the Consent of the Legislature of the State in which the Same shall be, for the Erection of Forts, Magazines, Arsenals, dock-Yards, and other needful Buildings;—And
-
-18: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and all other Powers vested by this Constitution in the Government of the United States, or in any Department or Officer thereof.
+13: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and which shall be within the scope of the foregoing Powers. It shall be the duty of all Representatives and Senators, the President of these United States, all State Executives, and State Legislatures, to ensure before their approval that a Bill under consideration falls within the scope of the foregoing powers, and is not in violation of any term of this Constitution.
 
 Section 9
-1: The Migration or Importation of such Persons as any of the States now existing shall think proper to admit, shall not be prohibited by the Congress prior to the Year one thousand eight hundred and eight, but a Tax or duty may be imposed on such Importation, not exceeding ten dollars for each Person.
+1: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it, whereupon it shall be suspended for more than one week only with the approval of three-fourths of both the House of Representatives and the Senate.
```

Comment:
Habeas Corpus is an incredibly important element of the law, and so it needs extreme guards. One week is not much time, but if it is not enough time to bring and announce charges against an imprisoned person, there needs to be a very convincing reason. Elected officials should be accountable for this.